ckanext/data_qld_theme/templates/package/comment_list.html

```diff
@@ -109,7 +109,7 @@ <h3>Reply to comment:</h3>
                         <h3>{{ comment.subject }}</h3>
                     {% endif %}
                 {% else %}
-                    <p><strong>{{ _('This comment was deleted.') }}</strong></p>
+                    {% snippet "snippets/comment_deleted.html", comment=comment %}
```

Comment:
does this file exist?

Reply:
it exists in the `ckanext-ytp-comments` extension here: https://github.com/MarkCalvert/ckanext-ytp-comments/blob/feature/DQL-188-dataset-comment-enhancements/ckanext/ytp/comments/templates/snippets/comment_deleted.html